README.md

```diff
@@ -1,10 +1,10 @@
-# Mobile menu
+# details popup
 A Microverse exercise to use animation on a website
```

Comment:
- [ ] Please I think you misunderstand the last reviewer on the javascript part she meant that you should loop over the array of objects and then you have to add the data to the list you have, instead of adding hard-coded index values to the list please use javascript to create the list of popup with there values.
![image](https://user-images.githubusercontent.com/74744787/232261108-d328519b-89a0-48e3-ad0b-e7978850bed4.png)

Reply:
I Understood but I will need too much time to do this. I was using my mind to do this project. I didn't know yet if I should follow this methodology.